notebooks/ovirs/part2/find-data-standalone.ipynb

```diff
@@ -10,7 +10,9 @@
     "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.            .filter("orex:Spatial.orex:target_range lt 4.0")                 \
</pre><p>Eventually we want to have methods in peppi to do that job. However we would have to specialize the peppi object for orex because the <code>orex:...</code> fields only exist for this mission.</p><p> </p><p>But that is fine for now.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NASA-PDS/search-api-notebook/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>